SETUP.md

```diff
@@ -71,69 +71,64 @@ Finally, flick the repository's switch to enable CI.
 Generate a deploy key so Travis CI can write to the repository.
 
 ```sh
+# IMPORTANT: change working directory to /ci
 cd ci
+
+# Generate deploy.key.pub (public) and deploy.key (private)
 ssh-keygen \
-  -t rsa \
-  -b 4096 \
-  -C "travis@travis-ci.com" \
-  -N "" \
+  -t rsa -b 4096 -N "" \
+  -C "deploy@travis-ci.com" \
   -f deploy.key
 
 # For convenience, print the URL to add the public key to GitHub
-echo https://github.com/$OWNER/$REPO/settings/keys
+echo "https://github.com/$OWNER/$REPO/settings/keys/new"
+
+# For convenience, print the URL to add the private key to Travis CI
+echo "https://travis-ci.com/$OWNER/$REPO/settings"
 ```
 
-Manually add the text of `deploy.key.pub` (with write access) to GitHub under the repository's deploy key settings (the URL echoed above).
-Give the key a descriptive title, such as "Travis CI Manubot".
+#### Add the public key to GitHub
 
-For the next step, you need the [Travis command line client](https://github.com/travis-ci/travis.rb) installed.
-This program is a Ruby gem:
-install it with `gem install travis` (not `apt install travis`, which is a different program).
-After the install, you will need to provide your credentials to login to travis with `travis login`.
+Go to the GitHub settings URL echoed above in a browser, and click "Add deploy key".
+For "Title", add a description like "Manubot Travis Deploy Key".
+Copy and paste the contents of the `deploy.key.pub` text file into the "Key" text box.
+Check the "Allow write access" box below.
+Finally, click "Add key".
 
-```sh
-travis encrypt-file \
-  --com \
-  --repo=$OWNER/$REPO \
-  --force \
-  deploy.key > travis-encrypt-file.log
-```
+#### Add the private key to Travis CI
 
-`ci/travis-encrypt-file.log` will contain a line such as:
+Next, go to the Travis CI repository settings page (URL echoed above).
+Add a new record in the "Environment Variables" section.
+For "NAME", enter `MANUBOT_SSH_PRIVATE_KEY`.
```

Comment:
Is `MANUBOT_SSH_PRIVATE_KEY` the best variable name? This verbose definition would be "a base64-encoded SSH private key to enable Travis CI to deploy commits to GitHub." @michaelmhoffman any intuition on the best variable name?

Reply:
Ticks the right boxes AFAICT.

Reply:
Agreed, `MANUBOT_SSH_PRIVATE_KEY` looks good to me.